patterns/event_bridge/src/main.rs

```diff
@@ -0,0 +1,76 @@
+use bevy::prelude::*;
+use boilerplate::*;
+
+mod boilerplate {
+    use bevy::ecs::event::Event;
+
+    use super::event_actor::ActorEvent;
+    use super::event_emitter::EmitterEvent;
+    
+    #[derive(Event, Clone)]
+    /// Glue for the different event types of the plugins
+    pub struct GameEvent(pub &'static str);
+
+    impl From<EmitterEvent> for GameEvent {
+        fn from(other: EmitterEvent) -> Self {
+            Self(other.0)
+        }
+    }
+
+    impl From<GameEvent> for ActorEvent {
+        fn from(other: GameEvent) -> Self {
+            Self(other.0.len())
+        }
+    }
```

Comment:
Not super sold on the `From`, instead of having a:
```mermaid
graph TD
  base --base event--> pluginA
  base --base event--> pluginB
  pluginA --> game
  pluginB --> game
```
we create this dependency abomination
```mermaid
graph TD
  base --base event--> pluginA
  base --base event--> pluginB
  pluginA --event A--> base
  pluginB --event B--> base
  pluginA --> game
  pluginB --> game
```